subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/radio/radio.c

```diff
@@ -725,7 +780,7 @@ u32_t radio_rssi_is_ready(void)
 void radio_filter_configure(u8_t bitmask_enable, u8_t bitmask_addr_type,
 			    u8_t *bdaddr)
 {
-	printk("%s\n", __func__);
+	/* printk("%s\n", __func__); */
```

Comment:
`BT_DBG("");` maybe?
Then you can enable debug on these using Kconfig.

Reply:
I understand your point of view. We have a number of functions (radio_filter_**) which we do not support and there we have printks commented out, so I wanted to be similar in approach. I cannot use BT_DBG because when activated it will print continuously and will block the normal operation (already tried it). If you think it's better, I can remove the printk both from this function and from the rest of the  radio_filter_** functions.

Reply:
Ok. Do what you think is best.